Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -1649,6 +1649,29 @@ bool RenderLayer::computeHasVisibleContent() const
     return false;
 }
 
+using LayerPositionAndSize = std::tuple<LayoutPoint, IntSize>;
```

Comment:
Can we just use a LayoutRect (and just know/assume that the size is integral)?

Reply:
Fair enough, it does simplify the code, so I'll adapt this as well before landing.